Always_green_CI.md

```diff
@@ -0,0 +1,56 @@
+
+# Concepts
+
+In this section is discussed the main pillars that sustain the "Always Green CI" initiative.
+
+## Baseline jobs
+
+There was created multiple jobs (called "baselines") which run on daily basis for the different test suites and configuration/HW which are identical to what will run on each pull request (PR). The key point is that this CI runs on a branch with no additional patches/noise added by anyone so if there is a failure it’s much easier to pinpoint it. The baseline jobs indicate the overall status of the Kata Containers project and we pursue to keep them "always green". They are used as a base for comparing with the CI jobs ran on each pull request (PR): if a given CI job fails but its corresponding on baseline is "green" then it is likely the code change breaking tests. 
+
+Each and every baseline CI job **must have** owners monitoring it’s status and acting if something goes wrong. Policies and a process to add/remove jobs to/from the baseline were defined.
+
+Find the complete list of baseline jobs [here](http://jenkins.katacontainers.io/view/Daily%20baseline/).
+
+## CI jobs
+
+Those are the jobs which will run on each pull request (PR). They are identical to baseline jobs, except that they test code changes proposed by the developers (the baseline jobs will test merged code). If a given CI job fails but its corresponding on baseline is "green" then it is likely the code change broke tests. So each and every CI job **must have** a counterpart baseline job.
+
+The CI job is configured to be either:
+
+ * required: meaning the job must pass for the PR be merged. This is reserved for jobs we are confident about their stability;
+ * non-required: its failure won't block the PR. This is reserved for jobs which are still unstable or/and experimental.
+
+The CI job is promoted to required if it has a proven record of stability over a period of time. Likewise it can be demoted to non-required in case it turned unstable. See the [guidelines section](#guidelines) on this document for further information about the policies and procedures to move a job from required to non-required (and vice-versa).
+
+Find the complete list of CI jobs [here](http://jenkins.katacontainers.io/view/PR%20Triggered/).
+
+## Owners of jobs
+
+Each and every baseline job **must have** a owner who is committed to maintain the stability of the job. Having co-owners working from different difficulties to pass their PR on the CI jobs
+
+# Guidelines
+
+## When to remove a job or add it back to the baseline list and CI?
+
+ * A baseline job is entitled for CI after 5 consecutive successes
+ * A job is marked non-required in the CI after 2 consecutive failures of its corresponding baseline job
+ * A job is removed from the baseline list after 15 consecutive failures
+ * A job can be marked back to required in the CI after 2 consecutive successes of its corresponding baseline job
```

Comment:
Nit: Numbers less than ten should really be spelt as words.

Reply:
Interesting...never thought about it. Thx! and Done.